docs/index.rst

```diff
@@ -18,13 +18,13 @@ customer datacenters.
 .. _Scality: https://www.scality.com
 .. _Zenko: https://www.zenko.io
 
-It is based on the Kubespray_ project to reliably install a base Kubernetes
+MetalK8s, based on the Kubespray_ project, reliably installs a base Kubernetes
```

Comment:
No. The intention is: it's based on Kubespray to reliably install a base Kubernetes cluster (which is Kubespray's job). The intent is not "MetalK8s reliably installs a base Kubernetes cluster".

Reply:
First of all, note the target audience of these docs is not only *users*, but also (potential) *contributors* and people who want to *understand* this project, so rationales make sense. This is not a 'Quickstart guide'.

To make it clear, MetalK8s is not *based* on Kubespray (which would make it a fork). Indeed, this is not correctly phrased in the original text, in retrospect.
It *includes* Kubespray to handle one part of what MetalK8s does (this part being 'installing a base Kubernetes cluster, reliably'). It also includes or deploys various other projects.

I fail to see, by the way, why the 'It' is ill-defined: the links in-between both paragraphs may throw you off, but this is not part of the rendered documents. In the rendered version (currently at https://metal-k8s.readthedocs.io/en/latest/) it seems quite clear to me what the 'It' refers to.

Reply:
It is clear to you, I trust, that the first word in this sentence is not a reference to a previously-defined term, but introduces the concept soon to follow. This structure and placement is present in the subject sentence. When you start a paragraph, you imply that you're starting with a fresh concept, not continuing the previous one. A reader's natural tendency is going to be "New paragraph. New idea. I may need to reset the 'its' buffer." That the "it" in question above refers to the aforementioned distribution becomes clear enough after one reads the sentence three times. I want to make it clear without having to slow the reader down. Fundamentally, this sentence lacks euphony and requires struggle to parse. Though you know exactly what you're describing, the reader has to pull in every clue available in realtime, and your sentence leaves a trail of breadcrumbs into the woods. 

Based on what you've told me, I suggest:.

> MetalK8s uses Kubespray, among other tools, to install a base Kubernetes cluster, including all dependencies (like etcd), using the Ansible provisioning tool. 

If you're wedded to "it", closing ranks with the preceding paragraph will help, but the sentence remains buggy. 

> It is based on the Kubespray project, and used for installing a base Kubernetes cluster, including all dependencies (like etcd), using the Ansible provisioning tool.